docs/README.md

```diff
@@ -56,18 +56,14 @@ work!](../CONTRIBUTING.md#sign-your-work)
 
 ## Branches
 
-**There are two branches related to editing docs**: `master` and `docs`. You
-should always edit the documentation on a local branch of the `master`
+You should always edit the documentation on a local branch of the `master`
 branch, and send a PR against `master`.
 
 That way your fixes will automatically get included in later releases, and docs
-maintainers can easily cherry-pick your changes into the `docs` release branch.
-In the rare case where your change is not forward-compatible, you may need to
-base your changes on the `docs` branch.
+maintainers can easily cherry-pick your changes into the `release` branch.
```

Comment:
We would like to find a better way of handling this, without cherry-picking onto the `release` branch.

/cc @crosbymichael @icecrime @tianon